### DISCUSSION TITLE
Add an option to the Low Detail plugin to disable people t-posing

### DISCUSSION THREAD

**Discussion by Arcuez** [runelite/runelite#16336] [Ideas]
Hey, could you add an option to the Low Detail plugin to stop people in crowded places from t-posing? I don't like how that looks and would like to keep using the plugin for the ground object removal feature.
![image](https://user-images.githubusercontent.com/59809669/222187162-6ab02931-f944-4ef5-a15b-ac58d8c1ec35.png)

**Comment by Nightfirecat** [comment]
FYI #12815 already exists for this request
